Integrations/integration-Tanium.yml

```diff
@@ -72,13 +75,41 @@ script:
     except Exception:
         raise
 
-    def csvstr_to_list(str):
-        lines = str.splitlines()
+    def parse_xml_to_csv(instring):
+        output = StringIO()
+        root = ET.fromstring(instring)
+        columns = []
+        for column in root.findall('./result_set/cs/c/dn'):
+            columns.append(column.text)
+        rows =[]
+        rows.append(columns)
+        for record in root.findall('./result_set/rs/r'):
+            values = record.findall('c/v')
+            # todo bug with count > 2 use limit=1 for  now in index file query
+            row = [i.text for i in values]
+            rows.append(row)
+        writer = csv.writer(output)
+        writer.writerows(rows)
+        return_string = output.getvalue()
+        return return_string
+
+    def csvstr_to_list(string):
+        lines = string.splitlines()
         if len(lines) < 2:
             return []
         else:
             headers = lines[0].split(',')
-            return [ dict(zip(headers, line.split(','))) for line in lines[1:] ]
+            new_headers= []
+            counter={}
+            for header_name in headers:
+                if header_name in counter:
+                    new_headers.append(header_name+str(counter[header_name]))
+                    counter[header_name]+=1
+                else:
+                    new_headers.append(header_name)
+                    counter[header_name]=0
```

Comment:
This way we are not getting multiple rows of data but rather expanding the current table, please fix this part.
We want the entries for different headers to be presented in a different row, and to be separated from the ones with the same name

Reply:
Ok so here lies the main issue that I have resolved.
If multiple columns have the same name only one column is injected into context via the EntryContext when using demisto.results and the remainder are ignored.    Thus the need to rename mulltiple columns with the same name.

As far as I can tell multiple rows are NOT returned when column names are the same.  It instead returns multiple columns with the same name. 

So yes If you would rather have multiple rows than multiple columns this entire PR can be closed as this is not what I did.

Reply:
I agree multiple rows would have been a better solution.  Let me know if that is the way you want to fix this and we can just close this PR.  I didn't do it that way.

Reply:
@shaniacht1 what's your take on this one, i believe it won't be a difficult solution to adjust the current solution into rows